app/src/main/res/values-ru/strings.xml

```diff
@@ -17,33 +17,33 @@
     <string name="lblScanExternal">Сканировать внешнее хранилище</string>
     <string name="lblNotificationMalwareDetectionTitle">Обнаружение вредоносного ПО</string>
     <string name="lblNotificationMalwareDetectionDescription">Используется для оповещения при обнаружении вредоносного ПО</string>
-    <string name="lblNotificationRealtimeTitle">Сканер в реальном времени</string>
+    <string name="lblNotificationRealtimeTitle">Онлайн-сканер</string>
     <string name="lblNotificationRealtimeDescription">Используется для отображения счетчика просканированных файлов и поддержки фоновой службы</string>
-    <string name="lblNotificationRealtimeText">Известные вредоносные программы будут обнаруживаться в режиме реального времени</string>
+    <string name="lblNotificationRealtimeText">Известное вредоносное ПО обнаруживается в режиме реального времени</string>
     <string name="lblNotificationRealtimeStopped">Hypatia: сканирование в реальном времени остановлено</string>
     <string name="lblNotificationRealtimeDetection">Обнаружено вредоносное ПО:</string>
-    <string name="lblRealtimeScannerToggle">Сканер в реальном времени</string>
+    <string name="lblRealtimeScannerToggle">Онлайн-сканер</string>
 
-    <string name="main_database_updating">Обновление баз данных %s...</string>
+    <string name="main_database_updating">Обновление %s баз данных...</string>
     <string name="main_database_downloading">Загрузка %s</string>
-    <string name="main_database_download_success">Успешно загружено</string>
+    <string name="main_database_download_success">Успешно загружена</string>
     <string name="main_database_download_error">Файл не загружен, код ответа %s</string>
     <string name="main_no_database_available">База данных недоступна, сканирование не выполняется...</string>
     <string name="main_database_released_on">Выпущены %s</string>
     <string name="main_database_not_changed">Файл не изменен</string>
-    <string name="main_database_not_modified_since">от %s</string>
+    <string name="main_database_not_modified_since">%s</string>
```

Comment:
removed?

Reply:
The old version definitely does not fit here - applications in Russian are not built that way.